docs/source/utils.rst

```diff
@@ -1,14 +1,18 @@
 .. _utils:
 
-torchvision.utils
-=================
+Utils
+=====
 
-.. currentmodule:: torchvision.utils
-
-.. autofunction:: make_grid
+The ``torchvision.utils`` module contains various utilities, mostly :ref:`for
+vizualization <sphx_glr_auto_examples_plot_visualization_utils.py>`. 
 
-.. autofunction:: save_image
+.. currentmodule:: torchvision.utils
 
-.. autofunction:: draw_bounding_boxes
+.. autosummary::
+    :toctree: generated/
+    :template: function.rst
 
-.. autofunction:: draw_segmentation_masks
+    draw_bounding_boxes
```

Comment:
And specific reason to change the order?
It's kinda nice to push `make_grid` down as we don't actively maintain it.

Reply:
>  It's kinda nice to push make_grid down as we don't actively maintain it.

That'st the reason :)